actors/miner/src/types.rs

```diff
@@ -482,3 +482,61 @@ pub struct GetBeneficiaryReturn {
 }
 
 impl Cbor for GetBeneficiaryReturn {}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct GetOwnerReturn {
+    pub owner: Address,
+}
+
+impl Cbor for GetOwnerReturn {}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct IsControllingAddressParam {
+    pub address: Address,
+}
+
+impl Cbor for IsControllingAddressParam {}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct IsControllingAddressReturn {
+    pub is_controlling: bool,
+}
+
+impl Cbor for IsControllingAddressReturn {}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct GetSectorSizeReturn {
+    pub sector_size: SectorSize,
+}
+
+impl Cbor for GetSectorSizeReturn {}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct GetAvailableBalanceReturn {
+    pub available_balance: TokenAmount,
+}
+
+impl Cbor for GetAvailableBalanceReturn {}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+pub struct GetVestingFundsReturn {
```

Comment:
Do we want `#[serde(transparent)]` here too? Also for GetPeerIDReturn and GetMultiaddrsReturn.

I suggest fixing with a PR to the source branch here, then propagating to next

Reply:
Will do, thank you.

Reply:
@anorth We discussed this (at least for GetPeerIDReturn) [here](https://github.com/filecoin-project/builtin-actors/pull/863/files#r1029652024). I think we should keep the slightly more complex types (such as GetVestingFundsReturn) to be serialized as tuples.

Reply:
Ack for PeerID/Multiaddrs.

I don't see logic for complexity of the single embedded object being a deciding factor about whether to wrap it in a tuple or not, tho

Reply:
Oh, the logic is entirely about consistency with existing types (cf. ComputeDataCommitmentReturn).